src/tests/TestWebServer/HttpServer.cs

```diff
@@ -100,17 +100,29 @@ public void Abort()
 
         private void Process(HttpListenerContext context)
         {
-            Logger.Info($"Handling request");
+            Logger.Info("Handling request {0}", context.Request.Url.AbsolutePath);
+
+            if (context.Request.Url.AbsolutePath == "/api/usage/unity")
+            {
+                context.Response.StatusCode = (int)HttpStatusCode.OK;
+                
+                var streamWriter = new StreamWriter(context.Response.OutputStream);
+                streamWriter.Write("Cool Unity usage bro!");
```

Comment:
The server returns json, not a string, we should return the same thing in the tests.